ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -282,9 +284,9 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
                                 dismissAnimated()
                             }
                         })
-                        .padding(.bottom, tokens.bottomPresentationPadding)
+                        .padding(.bottom, tokenSet[.bottomPresentationPadding].float)
                         .onSizeChange { newSize in
-                            bottomOffsetForDismissedState = newSize.height + (tokens.ambientShadowOffsetY / 2)
+                            bottomOffsetForDismissedState = newSize.height + (tokenSet[.ambientShadowOffsetY].float / 2)
```

Comment:
Super tiny nit #2: same as before, is there a reason this is /2, instead of just correct by itself?

Reply:
Well I don't know if I can because I do not know why!